docker-entrypoint.sh

```diff
@@ -1,21 +1,8 @@
 #!/bin/bash
 set -e
 
-if [ "$1" == 'bash' ]; then
-  exec bash
-elif [ "$1" == 'run' ]; then
-  BASE_CMD="uwsgi --master --manage-script-name --wsgi apiserver:app"
-  DEFAULT_ARGS="--socket /tmp/registry.sock --http 0.0.0.0:5030 --workers=2 --max-requests=10000"
-  if [ "$#" -gt 1 ]; then
-    echo "Running with arguments:" "${@:2}"
-    set -- ${BASE_CMD} ${@:2}
-  else
-    echo "Running with default arguments: ${DEFAULT_ARGS}"
-    set -- ${BASE_CMD} ${DEFAULT_ARGS}
-  fi
-else
-  echo 'Commands: run'
-  exit 1
+if [ "$1" = 'mywsgi' ]; then
```

Comment:
This is all the biggest change, but simplifies things drastically. There's no special case for `run` since that didn't even really make sense.

This took out all the specific uwsgi configs. The only things that were in here were `--workers=2` and `--max-requests=10000`, but I'd argue these are production specific things and we should bring those over to kube config as environment variables `UWSGI_PROCESSES=2` and `UWSGI_MAX_REQUESTS=10000` accordingly.

Side note, I assume 10,000 requests per worker is a lot for what this thing is and we might wanna lower it anyways.

Reply:
> Side note, I assume 10,000 requests per worker is a lot for what this thing is and we might wanna lower it anyways.

Could you elaborate on that?
FWIW, the running container is now stateless, and we aggressively cache all the responses (via Flask's SimpleCache).

Reply:
How much traffic does this get? I’m assuming not a lot.

Reply:
Not a lot at the moment.

Ah, you mean to lower the number of processes? Yes, we can do that.